Makefile

```
@@ -121,17 +121,23 @@ ACCEPTANCE_TIMEOUT:=20m
 .PHONY: acceptance
 
 acceptance: ## Run all acceptance tests
```

checkmake (0.3.2)

[warning] Target body for "acceptance" exceeds allowed length of 5 lines (17).

(maxbodylength)